raster_loader/io/snowflake.py

```diff
@@ -297,7 +302,11 @@ def band_rename_function(x):
 
             print("Writing metadata to Snowflake...")
             if append_records:
+                print("Appending records to Snowflake...")
+                print(f"Metadata: {metadata}")
```

Comment:
Metadata could be huge in some use cases, so printing it (new+old) is interesting always for debugging purposes but probably overwhelming for regular use. What do you think?

Reply:
Sorry, my bad. I was debugging the metadata to try to figure out what was going with the tests failing and didn't remove the prints. Now it's done.